docs/current_docs/manuals/developer/module-structure.mdx

```diff
@@ -421,26 +386,33 @@ use (
 </TabItem>
 <TabItem value="Python">
 
-Dagger modules in Python are built to be installed, like libraries. The runtime container installs the module code with:
+Dagger modules in Python are built to be installed, like libraries. At module creation time, a `pyproject.toml` and `uv.lock` file will automatically be created that depend on the locally generated client library (in `./sdk`). This dependency is configured to be editable so that changes in the code don't require a re-install.
+
+With the `uv.lock` file, Dagger uses uv's [project management](https://docs.astral.sh/uv/concepts/projects/) capabilites, but it can be opted out by removing this file, in which case Dagger falls back to [the pip interface](https://docs.astral.sh/uv/pip/) instead.
```

Comment:
```suggestion
With the `uv.lock` file, Dagger uses `uv`'s [project management](https://docs.astral.sh/uv/concepts/projects/) capabilites, but it can be opted out by removing this file, in which case Dagger falls back to [the `pip` interface](https://docs.astral.sh/uv/pip/) instead.
```

Reply:
As for "the pip interface", it's taken out of the linked page: https://docs.astral.sh/uv/pip/

Doesn't take backticks because it's not referring the `pip` CLI. It's a compatibility layer that pip has.